client/packages/common/src/intl/locales/en/common.json

```diff
@@ -31,6 +31,10 @@
   "button.select-a-color": "Select a colour",
   "button.try-again": "Try again",
   "button.view": "View",
+  "cold-chain.cold-consecutive": "Consecutive",
```

Comment:
Not sure if we want to say hot/cold with this since some mocks have it but some don't

Reply:
Maybe I'll add the `hot/cold` since you can't really tell in sensor list view 🤔